src/components/App.js

```diff
@@ -0,0 +1,20 @@
+import '../App.css';
+import React from 'react';
+import Display from './Display';
+import ButtonPanel from './ButtonPanel';
+
+const App = () => (
+  <div className="col-2 mx-auto">
```

Comment:
```
<>

<Display />

<ButtonPanel />

</>
```
React requires you to return all elements in one node so sometimes we create nodes that are not wanted, this requirement seeks to let you know the importance of [React Fragment](https://reactjs.org/docs/fragments.html).